docs/html/card/card--row.html

```diff
@@ -0,0 +1,16 @@
+<div class="ray-card ray-card--row" style="max-width: 400px;">
+  <div class="ray-card__image ray-image ray-image--16by9">
```

Comment:
Since this container won't conform to any aspect ratio (due to the fixed height/variable width), we'll probably want to use a bg here

Reply:
added!